TrueSDK/Core/Network/TCUrlProvider.m

```diff
@@ -0,0 +1,76 @@
+//
+//  TCUrlProvider.m
+//  TrueSDK
+//
+//  Created by Sreedeepkesav M S on 16/11/20.
+//  Copyright © 2020 True Software Scandinavia AB. All rights reserved.
+//
+
+#import "TCUrlProvider.h"
+
+@implementation TCUrlProvider
+
++ (NSURL *)baseUrlForCountryCode: (NSString *)countrycode {
+    if([[[self class] euCountryCodes] containsObject:countrycode]) {
+        return [NSURL URLWithString: @"https://outline-eu.truecaller.com"];
+    }
+    
+    return [NSURL URLWithString:@"https://outline-noneu.truecaller.com"];
```

Comment:
can this be done as extension/category of NSURL?

Reply:
Done